cordova-plugin-one/README.md

```diff
@@ -54,7 +54,22 @@ $ cordova plugin add cordova-plugin-one
 ## Configure the Thunderhead SDK
 To configure the ONE Cordova Plugin, declare a `window` variable and configure ONE when platform is ready. 
 * See example of usage [here](https://github.com/thunderheadone/one-sdk-cordova/tree/master/ionic5-angular-example/src/app/app.component.ts#L28)
-* To find your ONE credentials, see [Find the Information required when Integrating ONE with your Mobile App](https://na5.thunderhead.com/one/help/conversations/how-do-i/mobile/one_integrate_mobile_find_integration_info/)
+* The configuration parameters include:
+    * Site Key (for your specific Space)
+    * Touchpoint URI
+        * The native Thunderhead SDK will automatically prefix the URI scheme (i.e. android:// and ios://) when it is omitted. 
```

Comment:
might be clearer if it's explained: `The native Thunderhead SDK will automatically prefix the URI scheme, based on the platform the app runs on (i.e. `android://` and `ios://`), when it is omitted.  `

Reply:
Updated.